apps/documentation/src/content/docs/getting-started.md

```
@@ -0,0 +1,414 @@
+bun add <package> --filter @curiositi/server
+bun add <package> --filter @curiositi/web
+
+# Remove dependency
+bun remove <package> --filter @curiositi/server
+```
+
+## Troubleshooting
+
+### Database Issues
+
+**Error: relation "embedding" does not exist**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)